src/ReactiveDomain.Policy.Tests/PolicyMapTest.cs

```diff
@@ -1,27 +1,47 @@
 ﻿using ReactiveDomain.Messaging;
+using ReactiveDomain.Messaging.Bus;
+using ReactiveDomain.Testing;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Threading;
 using Xunit;
 
 namespace ReactiveDomain.Policy.Tests
 {
-    public class PolicyMapTest
+    public class PolicyMapTest :
+        IHandleCommand<AddItem>,
+        IHandleCommand<DisplayItem>,
+        IHandleCommand<DeleteItem>,
+        IHandleCommand<ExportItem>,
+        IHandleCommand<OtherCmd>,
+        IHandle<OtherMsg>
     {
-        [Fact]
-#pragma warning disable IDE1006 // Naming Styles
-        public void can_create_policy_map() {
-            var defaultPolicyName = "InteractiveUser";
-            var backupPolicyName = "AutomatedBackup";
-            var userRoleName = "User";
-            var adminRoleName = "Admin";
-            var customRoleName = "CustomRole";
-            var customPermissionName = "CustomPermission";
-            var backupRoleName = "Backup";
+        string defaultPolicyName = "InteractiveUser";
+        string backupPolicyName = "AutomatedBackup";
+        string userRoleName = "User";
+        string adminRoleName = "Admin";
+        string customRoleName = "CustomRole";
+        string customPermissionName = "CustomPermission";
+        string backupRoleName = "Backup";
+
+        UserDetails systemUser;
+        UserDetails adminUser;
+        UserDetails backupAgent;
 
-           //test and usage example
+        PolicyMap policyMap;
+        Role backupRole;
+
+        private long _addItem;
+        private long _displayItem;
+        private long _deleteItem;
+        private long _exportItem;
+        private long _otherCmd;
+        private long _otherMsg;
+
+        IDispatcher dispatcher;
+        private UserPolicy CurrentPolicy;
```

Comment:
Can we use the `_foo` naming convention for all of these fields for clarity?

Reply:
sure